dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -72,24 +72,30 @@ TaskFunction createFlutterViewStartupTest() {
   return new StartupTest(
       '${flutterDirectory.path}/examples/flutter_view',
       reportMetrics: false,
+      // This project has a non-standard CocoaPods Podfile. Run pod install
+      // before building the project.
+      customPodFile: true,
```

Comment:
I'd call this parameter `runPodInstall` to make it more obvious what the effect of this option is. It may not always be clear that custom POD file leads to an extra `pod install` command.